libhb/common.h

```diff
@@ -507,7 +507,9 @@ struct hb_job_s
 #define HB_VCODEC_FFMPEG_MPEG4 0x0000010
 #define HB_VCODEC_FFMPEG_MPEG2 0x0000020
 #define HB_VCODEC_FFMPEG_VP8   0x0000040
-#define HB_VCODEC_FFMPEG_VP9   0x0000080
+#define HB_VCODEC_FFMPEG_VP9   0x0000050
+#define HB_VCODEC_FFMPEG_H264  0x0000070
```

Comment:
I don't have time to do a full review at the moment.  But noticed this during a quick scan.

This should be HB_VCODEC_NVENC_H264 and HB_VCODEC_NVENC_H265.  "FFMPEG" is too generic.  FFMPEG_VP9 is a bad example to follow unfortunately, it should have been LIBVPX_VP9 or some such.